service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/NoticeOfChangeEmailContentProviderTest.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.notify.noticeofchange.NoticeOfChangeRespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.config.LookupTestConfig;
+
+import java.util.stream.Stream;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ContextConfiguration(classes = {NoticeOfChangeContentProvider.class, LookupTestConfig.class})
+public class NoticeOfChangeEmailContentProviderTest extends AbstractEmailContentProviderTest {
+
+    private static final Long CASE_ID = 12345L;
+    private static final String CASE_NAME = "Test";
+    public static final String FIRST_NAME = "John";
+    public static final String LAST_NAME = "Smith";
+    public static final CaseData CASE_DATA = CaseData.builder().id(12345L).caseName(CASE_NAME).build();
+
+    @Autowired
+    private NoticeOfChangeContentProvider underTest;
+
+
+    @ParameterizedTest
+    @MethodSource("solicitorNameSource")
+    void shouldReturnExpectedMapForSolicitorAccessGranted(String firstName, String lastName,
```

Comment:
ReturnExpectedMap does not sound right, method does not return map. Maybe shouldBuildNotificationData.. or something like ths